src/apps/ocioconvert/main.cpp

```diff
@@ -119,7 +119,7 @@ int main(int argc, const char **argv)
     std::cerr << "Loading " << inputimage << std::endl;
     try
     {
-        OIIO::ImageInput* f = OIIO::ImageInput::create(inputimage);
+        OIIO::ImageInput* f = (OIIO::ImageInput::create(inputimage)).get();
```

Comment:
OIIO's change to unique_ptr (from a raw ptr) is quite new. This change breaks the build with older versions of OIIO. You can use the OIIO_VERSION macro to conditionally handle both APIs.

Reply:
Cool, I'll look into it. Thx!